eveonline/forms.py

```diff
@@ -35,6 +35,9 @@ def clean(self):
                     raise forms.ValidationError(u'API key already exist')
                 if EveApiManager.api_key_is_valid(self.cleaned_data['api_id'], self.cleaned_data['api_key']) is False:
                     raise forms.ValidationError(u'API key is invalid')
+                if (settings.REJECT_OLD_APIS and
+                    EveManager.check_if_api_key_pair_is_new(self.cleaned_data['api_id'], settings.REJECT_OLD_APIS_MARGIN) is False):
+                    raise forms.ValidationError(u'API key is too old')
```

Comment:
Can this say something about requiring new API keys rather than just that it's too old?